### PR TITLE
Blob code tweaks

### PR DESCRIPTION
**Pull request by ChangelingRain**
Me, not knowing about or wanting to bother with merge conflict fixes the proper way!
Old pr #11555 

> Blob shields now actually block atmos thank me later
> Blob space expansion now always applies blob_act() even if it doesn't expand into the space tile, also it makes a sound if it fails so you know it's comin after ya

Makes blob code use a maxhealth var(health system 2 return of the health system)
Makes a var much less useful via health changes(this isn't a buff or a nerf it'll have the same results)
Blob health regen is now also a var.
Adds self to admins.txt

### CODE REVIEW

**Comment by Alek2ander** [comment]
The 1/4 power space tiles should only count as 1/4 of a tile for the blob victory counter.

**Comment by Anonus** [comment]
AH YES, MAGICAL CONFLICTS FROM ABSOLUTELYFUCKINGNOTHING

**Comment by Anonus** [comment]
FIXING CONFLICTS, WITH AN EMPTY COMMIT

**Comment by Anonus** [comment]
LIFE IS HELL, CONFIRMED

**Comment by PKPenguin321** [comment]
without oxygen blob *can* evoluate

**Comment by Anonus** [comment]
This is ready for merge!

**Comment by Anonus** [comment]
HELLO YES THIS IS STILL MERGE-READY PLEASE MERGE IT BEFORE IT CONFLICTS HARDER

File: code/game/gamemodes/blob/theblob.dm [hunk 1]
```diff
@@ -121,12 +121,15 @@
...
-	var/obj/effect/blob/normal/B = new /obj/effect/blob/normal(src.loc, min(src.health, 30))
+	var/obj/effect/blob/B = new /obj/effect/blob/normal(src.loc)
+	if(istype(T, /turf/space) && prob(65))
+		B.health = 0
 	B.color = a_color
 	B.density = 1
 	if(T.Enter(B,src))//Attempt to move into the tile
 		B.density = initial(B.density)
 		B.loc = T
+		B.update_icon()
```
Comment (phil235) [inline code/game/gamemodes/blob/theblob.dm]: Why do that change? What's bad about just doing `if(istype(T, /turf/space) && prob(65)) 	return` ?

File: code/game/gamemodes/blob/theblob.dm [hunk 2]
```diff
@@ -221,14 +224,23 @@
 	icon_state = "blob"
 	luminosity = 0
 	health = 21
+	maxhealth = 25
+	health_regen = 1
+	brute_resist = 4
```
Comment (phil235) [inline code/game/gamemodes/blob/theblob.dm]: that's quite a buff.

**Comment by phil235** [comment]
Instead of merging directly, I'm closing this and making a copy of this PR that I'll pull so there's less commits merged.

**Pull request closed without merging**